### ISSUE TITLE
go get throw me cannot call non-function t.torrent.Peers

### ISSUE THREAD

**Issue by MathieuDoyon** [Sioro-Neoku/go-peerflix#19]
```
github.com/Sioro-Neoku/go-peerflix/client.go:200: cannot call non-function t.torrent.Peers (type map[torrent.peersKey]torrent.Peer)
```

**Comment by danielcbit** [comment]
Got same problem yesterday. Building for Windows 10 32bit

**Comment by MathieuDoyon** [comment]
github.com/anacrolix/torrent has changed. Project should consider using vendor dependencies like godep, glide, etc.

client.go:200 `t.Peers()` should be without `()`

**Comment by CrazyLemon** [comment]
Already fixed in #17

**Comment by MathieuDoyon** [comment]
Sorry didn't saw.

Thanks
